test/uk/gov/hmrc/residencenilratebandcalculator/models/CalculatorTest.scala

```diff
@@ -16,126 +16,124 @@
       }
 
       "calculate the lost RNRB correctly if the former property was disposed after 8 July 2015 but before 6 April 2017, a 100000 RNRB Band applies (case study 15)" in {
-        Calculator.lostRnrb(new LocalDate(2019, 12, 10), new LocalDate(2016, 3, 21), 400000, 0, 105000) shouldBe Right(45000)
+        calculator.lostRnrb(new LocalDate(2019, 12, 10), new LocalDate(2016, 3, 21), 400000, 0, 105000) shouldBe Success(45000)
       }
 
       "calculate the lost RNRB correctly if there is no home in the estate (case study 16) " in {
-        Calculator.lostRnrb(new LocalDate(2021, 3, 3), new LocalDate(2018, 10, 5), 285000, 0, 0) shouldBe Right(175000)
+        calculator.lostRnrb(new LocalDate(2021, 3, 3), new LocalDate(2018, 10, 5), 285000, 0, 0) shouldBe Success(175000)
       }
 
       "calculate the lost RNRB as correctly if the value of the final property is less than the total amount of RNRB available (case study 17)" in {
-        Calculator.lostRnrb(new LocalDate(2021, 1, 10), new LocalDate(2019, 5, 21), 90000, 0, 0) shouldBe Right(105000)
+        calculator.lostRnrb(new LocalDate(2021, 1, 10), new LocalDate(2019, 5, 21), 90000, 0, 0) shouldBe Success(105000)
       }
 
       "calculate the lost RNRB correctly when there is transferred RNRB (case study 18)" in {
-        Calculator.lostRnrb(new LocalDate(2021, 3, 27), new LocalDate(2018, 10, 3), 285000, 175000, 0) shouldBe Right(332500)
+        calculator.lostRnrb(new LocalDate(2021, 3, 27), new LocalDate(2018, 10, 3), 285000, 175000, 0) shouldBe Success(332500)
       }
 
       "calculate the lost RNRB correctly when there is transferred RNRB and a final property" in {
         pending
         // Note that this test depends on the answer given to the long question asked in the comment in lostRNRB. If the
         // alternative form of teh calculation is used, this test should be added.
-        Calculator.lostRnrb(new LocalDate(2021, 3, 27), new LocalDate(2018, 10, 3), 285000, 175000, 90000) shouldBe Right(242500)
-      }
-
-      "given an error when the value of the former property is less than zero" in {
-        Calculator.lostRnrb(new LocalDate(2021, 1, 1), new LocalDate(2019, 1, 1), -1, 0, 0) shouldBe
-          Left(("INVALID_INPUTS", "The former property value must be greater or equal to zero."))
-      }
-
-      "given an error when the value of transferred RNRB is less than zero" in {
-        Calculator.lostRnrb(new LocalDate(2021, 1, 1), new LocalDate(2019, 1, 1), 100000, -1, 0) shouldBe
-          Left(("INVALID_INPUTS", "The transferred RNRB value must be greater or equal to zero."))
-      }
-
-      "given an error when the value of the final property is less than zero" in {
-        Calculator.lostRnrb(new LocalDate(2021, 1, 1), new LocalDate(2019, 1, 1), 100000, 0, -1) shouldBe
-          Left(("INVALID_INPUTS", "The value of the final property must be greater or equal to zero."))
```

Comment:
These tests are still valid and should be kept (though they may need to be updated as we now deal with a Try instead of Either)